src/org/ohdsi/usagi/ui/MappingDetailPanel.java

```diff
@@ -443,6 +444,11 @@ public void uncheckSelected() {
 		} else {
 			Global.mapping.fireDataChanged(MULTI_UPDATE_EVENT);
 		}
+		// If a row selected, then enable the add buttons
+		if (searchTable.getSelectedRow() != -1) {
+			replaceButton.setEnabled(true);
+			addButtons.forEach(x -> x.setEnabled(true));
```

Comment:
I notice that the buttons do not seem to get disabled anywhere. Not sure if that's intentional.

Reply:
They are disabled here:
https://github.com/OHDSI/Usagi/blob/e11d4b0eb75f73994768140d3d8385d2d7163c42/src/org/ohdsi/usagi/ui/MappingDetailPanel.java#L178-L181